Octokit.Tests.Integration/Clients/OrganizationHooksClientTests.cs

```diff
@@ -0,0 +1,197 @@
+
+                var hooks = await github.Organization.Hooks.GetAll( _fixture.org);
+
+                Assert.Single(hooks);
+                var actualHook = hooks[0];
+
+                AssertHook(_fixture.ExpectedHook, actualHook);
+            }
+        }
+
+        [Collection(OrganizationsHooksCollection.Name)]
+        public class TheGetMethod
+        {
+            readonly OrganizationsHooksFixture _fixture;
+
+            public TheGetMethod(OrganizationsHooksFixture fixture)
+            {
+                _fixture = fixture;
+            }
+
+            [IntegrationTest]
+            public async Task GetHookByCreatedId()
+            {
+                var github = Helper.GetAuthenticatedClient();
+
+                var actualHook = await github.Organization.Hooks.Get(_fixture.org, _fixture.ExpectedHook.Id);
+
+                AssertHook(_fixture.ExpectedHook, actualHook);
+            }
+        }
+
+        [Collection(OrganizationsHooksCollection.Name)]
+        public class TheCreateMethod
+        {
+            readonly OrganizationsHooksFixture _fixture;
+
+            public TheCreateMethod(OrganizationsHooksFixture fixture)
+            {
+                _fixture = fixture;
+            }
+            [IntegrationTest]
+            public async Task CreateAWebHookForTestOrganization()
+            {
+                var github = Helper.GetAuthenticatedClient();
+                var url = "http://test.com/example";
+                var contentType = OrgWebHookContentType.Json;
+                var secret = "53cr37";
+                var config = new Dictionary<string, string>
+                {
+                    { "url", "http://hostname.url" },
+                    { "content_type", "json" }
+                };
+                var parameters = new NewOrganizationHook("web", config)
+                {
+                    Events = new[] { "push" },
+                    Active = false
+                };
+
+                var hook = await github.Organization.Hooks.Create(_fixture.org, parameters.ToRequest());
+
+                var baseHookUrl = CreateExpectedBaseHookUrl(_fixture.org, hook.Id);
+                var webHookConfig = CreateExpectedConfigDictionary(config, url, contentType);
+
+                Assert.Equal("web", hook.Name);
+                Assert.Equal(new[] { "push" }.ToList(), hook.Events.ToList());
+                Assert.Equal(baseHookUrl, hook.Url);
+                Assert.Equal(baseHookUrl + "/pings", hook.PingUrl);
+                Assert.NotNull(hook.CreatedAt);
+                Assert.NotNull(hook.UpdatedAt);
+                Assert.Equal(webHookConfig.Keys, hook.Config.Keys);
+                //Assert.Equal(webHookConfig.Values, hook.Config.Values);
+                Assert.Equal(false, hook.Active);
+            }
+
+            Dictionary<string, string> CreateExpectedConfigDictionary(Dictionary<string, string> config, string url, OrgWebHookContentType contentType)
```

Comment:
Agree... also it seems like it isnt doing anything with the `url` and `contentType` parameters passed in